.github/workflows/ci.yml

```diff
@@ -10,15 +10,15 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - elixir_version: 1.16.1
-            otp_version: '26.2.1'
+          - elixir_version: 1.16.2
+            otp_version: '27.0'
```

Comment:
https://hexdocs.pm/elixir/main/compatibility-and-deprecations.html#between-elixir-and-erlang-otp

will wait for elixir 1.17

Reply:
rc is out